test/Constraints/fast-operator-typechecking.swift

```diff
@@ -14,7 +14,9 @@ func f(tail: UInt64, byteCount: UInt64) {
 }
 
 // rdar://problem/32547805
+/* FIXME(rxwei): This is broken by the introduction of the `Arithmetic` protocol.
```

Comment:
If you update `Policy.swift` to specify your new protocol in place of `Numeric` in the declarations of the operators that are now in that protocol, this will return to type checking quickly.